src/test/kotlin/com/hibob/academy/service/UserServiceTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.hibob.academy.service
+
+import com.hibob.bootcamp.unittests.*
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.kotlin.any
+
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+
+class UserServiceTest {
+    private val userDao = mock<UserDao>{}
+    private val notificationService = mock<NotificationService>{}
+    private val emailVerificationService = mock<EmailVerificationService>{}
+    private val userService = UserService(userDao, notificationService, emailVerificationService)
+
+    @Test
+    fun `register null user and throws exception`() {
```

Comment:
The test name is not describing what is actually tested, should be:
`registerUser should throw exception if user already exists`